[DOCS] Updates introduction doc (#108122)
elastic/kibana: docs/user/introduction.asciidoc

*{kib} works with all types of data.* Your data can be structured or unstructured text,

> numerical data, time-series data, geospatial data, logs, metrics, security events,

Reviewer: This is one of the terms in https://brand.elastic.co/302f66895/p/194a3b-writing-style-guide
```suggestion
numerical data, time series data, geospatial data, logs, metrics, security events,
```

* Roll up data from one or more indices into a new, compact index.
* Replicate indices on a remote cluster and copy them to a local cluster.

For a full list of data management UIs, refer to <<management, *Stack Management*>>.

[role="screenshot"]
image::images/stack-management.png[Index Management view in Stack Management]

[float]
=== Alert and take action

Detecting and acting on significant shifts and signals in your data is a need
that exists in almost every use case. Alerting allows you to
detect conditions in different {kib} apps and trigger actions when those conditions are met.
For example, you might trigger an alert when a shift occurs in your business critical KPIs or when
memory, CPU, or disk space take a dip.
When the alert triggers, you can send a notification to a system that is part of your daily workflow:
email, Slack, PagerDuty, ServiceNow, and other third party integrations.

A dedicated view for creating, searching,
and editing rules is in <<create-and-manage-rules,*Rules and Connectors*>>

[role="screenshot"]
image::images/rules-and-connectors.png[Rules and Connectors view]

[float]
[[organize-and-secure]]
=== Organize content

> You might be managing tens, hundreds, or even thousands of dashboards, visualizations, and other Kibana assets.

Reviewer 2: ```suggestion
You might be managing tens, hundreds, or even thousands of dashboards, visualizations, and other {kib} assets.
```
```suggestion
You might be managing tens, hundreds, or even thousands of dashboards, visualizations, and other Kibana assets.
```

{kib} has several features for keeping your content organized.

[float]
[[organize-in-spaces]]
==== Collect related items in a space

{kib}

> <<xpack-spaces, Spaces>> is built for organizing your visualizations, dashboards, and index patterns.

Reviewer 2: ```suggestion
<<xpack-spaces, Spaces>> is built for organizing your visualizations, dashboards, and {data-sources}.
```
```suggestion
<<xpack-spaces, Spaces>> is built for organizing your visualizations, dashboards, and index patterns.
```

`type:(dashboard or canvas-workpad) logs` +
|===

This example searches for visualizations with the tag `design` .

[role="screenshot"]
image::images/tags-search.png[Example of searching for tags]

[float]
=== View all {kib} has to offer

> To view the full list of {kib} apps and features, go https://www.elastic.co/kibana/features[{kib} features].

Reviewer 2: ```suggestion
To view the full list of {kib} apps and features, go to https://www.elastic.co/kibana/features[{kib} features].
```
```suggestion
To view the full list of {kib} apps and features, go https://www.elastic.co/kibana/features[{kib} features].
```

[float]
[[try-kibana]]
=== Get help

> Click the help icon image:images/intro-help-icon.png[Help icon in navigation bar]

Reviewer 2: ```suggestion
Click image:images/intro-help-icon.png[Help icon in navigation bar]
```
```suggestion
Click the help icon image:images/intro-help-icon.png[Help icon in navigation bar]
```
